docs/documentation.md

```diff
@@ -13,6 +13,11 @@ All the commands in this document are assumed to be run from the `/docs` folder,
 1. Install `Node.js` (minimum version 12).
 1. Run `npm ci` to install the necessary tools to build documentation, including MarkBind.
 
+Install the following additional dependencies required by MarkBind to generate [PlantUML](https://plantuml.com/) diagrams locally:
+
+1. Install [Java](https://www.java.com/en/download/) 8 (or later).
```

Comment:
No need to add link here. Java is a general tool, and we don't want to give the wrong impression of being constrained to use Oracle JDK. (e.g. I myself use Amazon Corretto)
```suggestion
1. Install Java 8 or later.
```